src/averageAges.js

```diff
@@ -35,7 +54,28 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
-}
+
+  const arrCorrected = (people.map(person => {
+    const collectPerson = [];
+    const isMother = (persons) => people.some((human) => human.mother
```

Comment:
```suggestion
    const isMother = (person) => people.some((human) => human.mother
```
person - це один
people - багато